src/basic/terminal-util.c

```diff
@@ -1278,6 +1278,9 @@ ColorMode get_color_mode(void) {
                         /* We only check for the presence of the variable; value is ignored. */
                         cached_color_mode = COLOR_OFF;
 
+                else if (getenv("COLORTERM"))
```

Comment:
Hm? If I believe your comment, it has "truecolor" or "24bit". Why not to parse it and set it to COLOR_16777216 or COLOR_24bit?

Reply:
Agreed

Reply:
Hmm, you said "agreed", but the variable is still not being parsed ??

Reply:
Needed an input here actually but did not ask. When parsed, i need validate it against "truecolor" and "24bit", right?

Reply:
Adding now.